Source/FullScreenPokemon.ts

```diff
@@ -10044,7 +10068,18 @@ module FullScreenPokemon {
         }
 
         /**
+         * Function to add a Key-Value pair to an Array. If the pair already exists,
```

Comment:
Lower-case k in Key and lower-case v in Value (general terms, not data types). Also, that's not a very accurate term for this.

Rename it to something "countable container".